v1.0/create-security-certificates.md

```diff
@@ -96,106 +96,144 @@ If you need to troubleshoot this command's behavior, you can change its [logging
 
 ### Create the CA certificate and key pair
 
-1. Create a safe directory for the CA key:
+1. Create two directories:
 
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ mkdir certs
+    ~~~
+
+    {% include copy-clipboard.html %}
     ~~~ shell
     $ mkdir my-safe-directory
     ~~~
+    - `certs`: You'll generate your CA certificate and all node and client certificates and keys in this directory and then upload the files to your nodes.
+    - `my-safe-directory`: You'll generate your CA key in this directory and then reference the key when generating node and client certificates. After that, you'll keep the key safe and secret; you will not upload it to your nodes.
 
 2. Generate the CA certificate and key:
 
+    {% include copy-clipboard.html %}
     ~~~ shell
     $ cockroach cert create-ca \
     --certs-dir=certs \
     --ca-key=my-safe-directory/ca.key
+    ~~~
 
+    {% include copy-clipboard.html %}
+    ~~~ shell
     $ ls -l certs
     ~~~
 
     ~~~
-    -rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
+    total 8
+    -rw-r--r--  1 maxroach  maxroach  1.1K Jul 10 14:12 ca.crt
     ~~~
 
 ### Create the certificate and key pairs for nodes
 
-1. Create the certificate and key for the first node:
+1. Generate the certificate and key for the first node:
 
+    {% include copy-clipboard.html %}
     ~~~ shell
     $ cockroach cert create-node \
     node1.example.com \
     node1.another-example.com \
     --certs-dir=certs \
     --ca-key=my-safe-directory/ca.key
+    ~~~
 
+    {% include copy-clipboard.html %}
+    ~~~ shell
     $ ls -l certs
     ~~~
 
     ~~~
-    -rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
-    -rw-r--r--  1 maxroach  maxroach  1.2K Apr 19 09:36 node.crt
-    -rw-------  1 maxroach  maxroach  1.6K Apr 19 09:36 node.key
+    total 24
+    -rw-r--r--  1 maxroach  maxroach  1.1K Jul 10 14:12 ca.crt
+    -rw-r--r--  1 maxroach  maxroach  1.2K Jul 10 14:16 node.crt
+    -rw-------  1 maxroach  maxroach  1.6K Jul 10 14:16 node.key
     ~~~
 
 2. Upload the files to the first node.
```

Comment:
uh. both `upload the files to the Nth node` steps are empty in this doc. Is it because this is an overview of how certs work? It seems odd though.